packages/date-picker/test/keyboard-input.test.js

```diff
@@ -129,32 +129,6 @@ describe('keyboard', () => {
     });
   });
 
-  describe('no parseDate', () => {
```

Comment:
These tests were added in https://github.com/vaadin/web-components/pull/3372, though it's not clear what the issue was. With the changes here it should be impossible to have no `parseDate` function so it seems safe to remove?